src/algebra/big_operators/basic.lean

```diff
@@ -253,6 +253,21 @@ end
 
 end comm_monoid
 
+section to_list
+
+open finset
+
+@[to_additive]
+lemma prod_to_list [comm_monoid β]
```

Comment:
Why not put this lemma in the `comm_monoid` section above?

Reply:
can do, but wouldn't it be better to have a section named `to_list` dedicated to stuff about `finset.to_list`?

Reply:
You can always put that section inside the `comm_monoid` section. Do you expect to have more than one lemma though anyway?